layouts/event/single.html

```diff
@@ -0,0 +1,98 @@
+{{ define "main" }}
+{{ if .Params.license }}
+{{ $.Scratch.Set "license" .Params.license }}
+{{ else }}
+{{ $.Scratch.Set "license" "cc-by-4.0"}}
+{{ end }}
```

Comment:
The license isn't actually displayed anywhere. In the interest of getting this merged asap, I won't block on this but it should be addressed.